### PR TITLE
Mixer-21 improve: Crossfade button enhancement and positioning

### LINKED ISSUE #21: Crossfade buttons enhancement
Select a time and then press the crossfade to actually crossfade.

### PR DESCRIPTION
**Pull request by Froopity**
Closes #21

### CODE REVIEW

File: src/app/component/crossfade-buttons/crossfade-buttons.component.html [hunk 1]
```diff
@@ -1,4 +1,5 @@
-<button (click)="func(1)">1</button>
-<button (click)="func(5)">5</button>
-<button (click)="func(10)">10</button>
-<button (click)="func(30)">30</button>
+<button *ngFor="let duration of durations" (click)="click(duration)"
+        class="btn" [ngClass]="isSelected(duration) ? 'btn-selected' : 'btn'">
```
Comment (DijitGuy) [inline src/app/component/crossfade-buttons/crossfade-buttons.component.html]: remind me what `?` does in this context again?

**Comment by DijitGuy** [comment]
looks good boss

**Comment by Froopity** [comment]
let x;
if (bool) {
  x = 'yay';
} else {
  x = 'nay';
}

Is equivalent to

x = bool ? 'yay' : 'nay'

It's called the ternary operator

File: src/app/component/crossfade-buttons/crossfade-buttons.component.html (hunk 1 above)
Comment (Froopity) [reply src/app/component/crossfade-buttons/crossfade-buttons.component.html]: let x;
if (bool) {
  x = 'yay';
} else {
  x = 'nay';
}

Is equivalent to

x = bool ? 'yay' : 'nay'

It's called the ternary operator

**Pull request merged**